examples/lighting-app/nrfconnect/README.md

```diff
@@ -1,302 +1,344 @@
+        SDK source directory.
+    -   _~/connectedhomeip_ must be replaced with an absolute path to the CHIP
+        source directory.
+    -   _-v /dev/bus/usb:/dev/bus/usb --device-cgroup-rule "c 189:_ rmw"\*
+        parameters can be omitted if you are not planning to flash the example
+        onto hardware. These parameters give the container access to USB devices
+        connected to your computer such as the nRF52840 DK.
+    -   _--rm_ can be omitted if you do not want the container to be
+        auto-removed when you exit the container shell session.
+    -   _-e RUNAS=\$(id -u)_ is needed to start the container session as the
+        current user instead of root.
 
-Now you may build the example by running the commands below in the Docker
-container:
+4.  Check out or update the nRF Connect SDK to the recommended `v1.4.0` version
+    by running the following command in the Docker container:
 
-        $ cd /var/chip/examples/lighting-app/nrfconnect
-        $ west build -b nrf52840dk_nrf52840
+         $ setup --ncs v1.4.0
+         /var/ncs repository is empty. Do you wish to check out nRF Connect SDK sources [v1.4.0]? [Y/N] y
+         ...
+         /var/chip repository is initialized, skipping...
 
-If the build succeeds, the binary will be available under
-`/var/chip/examples/lighting-app/nrfconnect/build/zephyr/zephyr.hex`. Note that
-other operations described in this document like flashing or debugging can also
-be done in the container.
+Now you can proceed with the [Building](#building) instruction.
 
-### Using native shell
+### Using native shell for setup
 
-Before building the example,
-[download the nRF Connect SDK and install all requirements](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_installing.html).
-Please read the
-[Supported nRF Connect SDK versions](#supported-nrf-connect-sdk-versions)
-section to learn which version to use to avoid unexpected compatibility issues.
+To use the native shell for setup, complete the following steps:
 
-If you don't want to use SEGGER Embedded Studio, you may skip the part about
-installing and configuring it.
+1.  Download and install the following additional software:
 
-Download and install the
-[nRF Command Line Tools](https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF-Command-Line-Tools).
+    -   [nRF Command Line Tools](https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF-Command-Line-Tools)
+    -   [GN meta-build system](https://gn.googlesource.com/gn/)
 
-Download and install [GN meta-build system](https://gn.googlesource.com/gn/).
+2.  Depending on whether you have the nRF Connect SDK installed:
 
-Make sure that you source the following file:
+    -   Follow the
+        [guide](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_assistant.html#)
+        in the nRF Connect SDK documentation to install the nRF Connect SDK
```

Comment:
So this points to using the gui tool to install the SDK, right?  It'd be nice if there were just pointers here to the right things to download, which can be derived, with some work, from the links that used to be here...

Reply:
Have you tried the GUI tool (Getting Started Assistant)? Did you find any issues using it? We considered several options for this section. The article which used to be linked here, which describes manual steps for the setup, was unclear for some devs as it refers to other articles in the Zephyr documentation etc. and it is quite easy to get lost. Listing the steps in this README directly would require us to cover multiple operating systems and not to assume that a user has any software installed. We thought Getting Started Assistant might provide less error-prone solution for setting up the environment as it verifies all steps made so far. But let me know your thoughts on that so we may do another round of the documentation improvement :)

Reply:
I haven't yet, but in the past I've had my share of trouble with some tools when I needed to run a build on a system I only had console access to....